docs/Intro.md

```diff
@@ -1,48 +1,48 @@
 
 So go do it! Go! Go [make an app](https://discord.com/developers/applications) and create something awesome.
 
 ## Games
 
 We love games, and we love helping game developers. If YOU have a game that you want to supercharge with Discord, then we have just the thing for you.
 
-#### Your Game and Discord
+- #### Your Game and Discord
 
-We’ve got a whole suite of tools to help bring your game to life with Discord. Need networking? Have [networking](#DOCS_GAME_SDK_NETWORKING/)! Want a friends list? Take [relationships](#DOCS_GAME_SDK_RELATIONSHIPS/) too!
+We’ve got a whole suite of tools to help bring your game into life with Discord. Need networking? Have [networking](#DOCS_GAME_SDK_NETWORKING/)! Want a friends list? Take [relationships](#DOCS_GAME_SDK_RELATIONSHIPS/) too!
 
-#### Rich Presence
+- #### Rich Presence
 
-No matter where your game lives, it should have a first-class experience in Discord. [Rich Presence](https://discord.com/rich-presence) is the way to make that a reality! We got tired of exchanging usernames, friend codes, and lobby passwords, so we created Rich Presence, an easy-to-use, easy-to-integrate way to get people playing games together that lets you:
+No matter where your game lives, it should have a first-class experience in Discord. [Rich Presence](https://discord.com/rich-presence) is the way to make that a reality! We had got tired of exchanging usernames, friend codes, and lobby passwords, so we created Rich Presence, an easy-to-use, easy-to-integrate way to get people play games together that lets you:
 
 - Display rich game data on your players’ profiles
 - Empower them to send game invites to each other
 - Ask to Join and Spectate their friends' games
 - Spend more time playing together and less time setting up
 
-It’s free, easy, and self-serve, so check out [the GameSDK Activity Manager](#DOCS_GAME_SDK_ACTIVITIES/) and get started!
+It’s free, easy to use, and self-serve, so check out the [GameSDK Activity Manager](#DOCS_GAME_SDK_ACTIVITIES/) and get started!
 
 ## Still need some help?
 
-Join the [Official Discord Developers server](https://discord.gg/discord-developers) for support and discussion regarding Discord's APIs.
+Join the [Official Discord Developers Server](https://discord.gg/discord-developers) for support and discussion regarding Discord's APIs.
 
 ## Go Make Cool Stuff!
 
-We love our developers, and we plan to keep making practical tools so that YOU can keep making cool stuff. Build a bot, integrate our account system, or put us right in your game; whatever you do, do it with Discord.
+We love our developers, and we plan to keep making practical tools so that YOU can keep making cool stuff for everyone. Build a bot,create an api wrapper, integrate our account system, or put us right in your game; or whatever you do, do it with Discord.
```

Comment:
```suggestion
We love our developers, and we plan to keep making practical tools so that YOU can keep making cool stuff for everyone. Build a bot, create an API wrapper, integrate our account system, or put us right in your game; whatever you do, do it with Discord.
```